_impls/apache-sshd.md

```diff
@@ -6,49 +6,66 @@ license: "[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0)"
 first-release:
     date: 2009      # according to Wikipedia
 latest-release:
-    version: 1.2.0
-    date: 2016-03-09
-changelog: "https://issues.apache.org/jira/browse/SSHD/?selectedTab=com.atlassian.jira.jira-projects-plugin:changelog-panel"
+    version: 2.4.0
+    date: 2020-01-17
+changelog: "https://github.com/apache/mina-sshd/blob/master/CHANGES.md"
 client: yes
 server: yes
 library: both
 protocols:
     cipher:
-        - aes128-ctr
-        - aes192-ctr
-        - aes256-ctr
-        - arcfour256
+        - aes128ctr
+        - aes192cbc
+        - aes192ctr
+        - aes256cbc
+        - aes256ctr
         - arcfour128
-        - aes128-cbc
-        - 3des-cbc
-        - blowfish-cbc
-        - aes192-cbc
-        - aes256-cbc
+        - arcfour256
+        - blowfishcbc
+        - tripledescbc
     compression:
         - zlib
         - zlib@openssh.com
         - none
     hostkey:
-        - ssh-rsa
         - ssh-dss
-        - ecdsa-sha2-nistp256
-        - ecdsa-sha2-nistp384
-        - ecdsa-sha2-nistp521
```

Comment:
Are you sure those were removed? There are still tests for them in the mina-sshd repository